mlflow/gateway/fluent.py

```diff
@@ -56,11 +56,12 @@ def create_route(name: str, route_type: str, model: Dict[str, Any]) -> Route:
         route configuration is handled via updates to the route configuration YAML file that
         is specified during Gateway server start.
 
-    :param name: The name of the route.
+    :param name: The name of the route. This parameter is required for all routes.
     :param route_type: The type of the route (e.g., 'llm/v1/chat', 'llm/v1/completions',
-                       'llm/v1/embeddings').
+                       'llm/v1/embeddings'). This parameter is required for routes that are
+                        not managed by Databricks (the provider isn't 'databricks').
     :param model: A dictionary representing the model details to be associated with the route.
-                  This dictionary should define:
+                  This parameter is required for all routes. This dictionary should define:
```

Comment:
Do we still need `This parameter is required for all routes.`?

Reply:
I think it's still good to call it out in the docs so the customer isn't surprised at the error. WDYT?

Reply:
What if we add a new provider in Databricks that doesn't require `model`?

@BenWilson2 Thoughts?

Reply:
The client should be consistent with the API. If the model is a required field for the API, then the client should document that. It's a bit odd for `model=None` to be required, but that's fine for now. We can fix this later.